indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -130,6 +154,6 @@ def as_legacy(self):
         amgr = self.event_new.as_legacy.getAbstractMgr()
         return amgr.getAbstractById(str(self.friendly_id))
 
-    @property
-    def accepted_track(self):
-        return self.event_new.as_legacy.getTrackById(str(self.accepted_track_id))
+    @return_ascii
+    def __repr__(self):
+        return format_repr(self, 'id', friendly_id=self.friendly_id)
```

Comment:
The title is missing (`_text=..`). Actually, it's also missing in the model itself..